x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/fields/code_editor.tsx

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
Pulled over from legacy uptime, with a few adjustments to make it work with react-hook-form.